guides/source/security.md

```diff
@@ -436,6 +436,135 @@ send_file filename, disposition: "inline"
 
 Another (additional) approach is to store the file names in the database and name the files on the disk after the ids in the database. This is also a good approach to avoid possible code in an uploaded file to be executed. The `attachment_fu` plugin does this in a similar way.
 
+Authentication
+--------------
+
+Rails 8 introduces an authentication generator, which is meant to be a solid starting point for securing your application by only allowing access to verified users.
```

Comment:
We generally avoid mentioning a version in the docs, so I can remove the Rails 8 part. Though I felt like clarifying in this case as having "authentication" in Rails is a "big" shift. I'll probably just say "Rails" though.